pkg/client/unversioned/client.go

```diff
@@ -97,6 +98,10 @@ func (c *Client) Secrets(namespace string) SecretsInterface {
 	return newSecrets(c, namespace)
 }
 
+func (c *Client) ConfigDatas(namespace string) ConfigDatasInterface {
```

Comment:
I have the same problems with the naming as well. It sounds a bit awkward, but I chose it to preserve the conventions used in the rest of the resources.